powerauth-push-server/src/main/java/io/getlime/push/service/batch/storage/ItemStorage.java

```diff
@@ -21,7 +21,7 @@
  * @param <K> Key class for item storage.
  * @param <V> Value class for item storage.
  */
-public interface ItemStorageMap<K,V> {
+public interface ItemStorage<K,V> {
```

Comment:
The interface should not say much about the underlying implementation. Anyway, we should switch to `Cache` later on instead of the custom one, to enable such a feature as `#get(K key, Function<K, V> mappingFunction)`.